bot/tests/not_sending_videos/test_episode_list.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from bot.tests.base_test import BaseTest
+
+
+class TestEpisodesListsCommand(BaseTest):
+
+    @pytest.mark.quick
+    def test_episodes_for_valid_season(self):
+        response = self.send_command('/odcinki 4')
+        expected_fragments = ["Szok poporodowy","Zemsta i wybaczenie"]
+        self.assert_response_contains(response, expected_fragments)
+
+    @pytest.mark.quick
+    def test_episodes_for_nonexistent_season(self):
+        response = self.send_command('/odcinki 99')
+        expected_fragments = ["❌ Nie znaleziono odcinków dla sezonu 99."]
+        self.assert_response_contains(response, expected_fragments)
```

Comment:
to już mi się mocno nie podoba, że te testy zależą stricte od rancza, powinny być lekkie testowe assety, a nie ściągaj 130GB rancza czy ileś i chuj żeby puścić test na listę elementów xDDD

Reply:
Dosłownie możesz jakieś losowe parę filmików zrobić i transkrypcje na pałę, żeby miały sens programatyczny, a niekoniecznie użytkownikowy

Reply:
A e2e to nie jest dosłownie kopia apki i co co cuduje usera ?

Reply:
Generalnie tak, ale skoro planujesz dodawac inne seriale, to warto miec testy ktore testuja calego bota

Nie potrzebujesz poprawnej transkrypcji rancza zeby przetestowac kiepskich, a natura tego bota jest taka, ze poprawnosc transkrypcji i tak musisz recznie sprawdzac

To co mozna robic automatycznie to "zakladajac transkrypcje X dostane wynik Y", po co sobie dokladac roboty jakbys chcial ja z jakiegos powodu poprawic na produkcji? A ta smieciowa jaka byla taka bedzie + lzejsze srodowisko testowe

Reply:
może i tak tylko ta komenda /odcinki jest taka że i tak jakiś jeden sezon odcinków czegoś muszę wrzucić żeby miała sens i to o strukturze takiej jaka jest w transkrypcjach ranczowych a o komnedzie specjalnej /odcinki 11 nie wspomnę XD

Reply:
wgl tak sobie myślę czy specjalnych komend nie dało by się wydzielić do jakiegoś osobnego middleware czy czegoś żeby bot był uniwersalny i tylko jakiś jego config czy coś napisywał jakaś komenda wartością specjalna albo dodawał specjalne komendy czy coś takiego

Reply:
to jakbyś chciał to uniezależnić od rancza bo jak rozumiem e2e musi testować wszystko czyli ja jakieś pliki jsona musze wyindexowac w elasticu żeby po nich szukać to pytanie po co pierdolić w robienie jakiś custom czy coś skoro te z rancza są rdy to use

Reply:
> wgl tak sobie myślę czy specjalnych komend nie dało by się wydzielić do jakiegoś osobnego middleware czy czegoś żeby bot był uniwersalny i tylko jakiś jego config czy coś napisywał jakaś komenda wartością specjalna albo dodawał specjalne komendy czy coś takiego

o tym juz gadalismy, wszystko ogolne tutaj (opcjonalnie sterowane envami/konfigami), a w razie czego fork dla konkretnego serialu gdzie dodajesz nowe rzeczy tak jakbys dodawal child class

Reply:
> to jakbyś chciał to uniezależnić od rancza bo jak rozumiem e2e musi testować wszystko czyli ja jakieś pliki jsona musze wyindexowac w elasticu żeby po nich szukać to pytanie po co pierdolić w robienie jakiś custom czy coś skoro te z rancza są rdy to use

znaczy no idea byla taka, zeby to bylo mozliwie lekkie, a zakladam ze caly sezon rancza swoje wazy xDD ale no jak to wymaga duzo roboty zeby jakis smieciowy input zrobic to ig technical debt i lec z ranczem xD